Assets/BossRoom/Scripts/Game/Client/UI/ConnectionStatusMessageUIManager.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using Unity.Multiplayer.Samples.BossRoom.Shared.Infrastructure;
 using UnityEngine;
+using VContainer;
```

Comment:
needed?

Reply:
Yup, to provide [Inject] attribute, now that we're using VContainer

Reply:
But I don't see any [Inject] in this file though...
Is this a codegen thing? Could you add comments to these usings so users understand this please?

Reply:
There is one for the InjectDependencies() method below, no?

Reply:
oooh now I understand, since it's using the same "Inject" name, there's no new changes to this file.
Now the question is, why aren't we doing like the others and injecting directly in properties?

Reply:
Because there's logic that needs to be run after we get the dependencies injected - specifically, there are some subscriptions happening. True, one can do the same by using VContainer interfaces for the IStartable or whatnot, but... why? Same result, tom[ay]to==tom[ah]to